readthedocs/api/v3/tests/responses/projects-builds-detail.json

```diff
@@ -10,6 +10,11 @@
         "project": "https://readthedocs.org/api/v3/projects/project/",
         "version": "https://readthedocs.org/api/v3/projects/project/versions/v1.0/"
     },
+    "urls": {
+        "build": "https://readthedocs.org/projects/project/builds/1/",
+        "project": "https://readthedocs.org/projects/project/",
+        "version": "https://readthedocs.org/dashboard/project/version/v1.0/"
```

Comment:
The project and version URL are going to be the same on every element of the list, not sure about that (we do the same in _links already). I think maybe just add the build url for now, and see if we need the others later.

Reply:
Okay sure! updated :)